### ISSUE TITLE
Time for another rebase?

### ISSUE THREAD

**Issue by brookr** [patshaughnessy/paperclip#7]
It's been about a year... Time to sync up with the parent project again?

Thanks for your work on this helpful project.

**Comment by patshaughnessy** [comment]
Thanks Brook... yup you're right, it's time. I'll do my best to get to it in December... on my list to do.

**Comment by brookr** [comment]
Rockin'! That'd be a great xmas gift for us all. Thanks a ton. ;]

**Comment by wleeperogs** [comment]
Doesn't work with rails3.2.rc1, I know it's only the first day, but maybe a rebase will help

**Comment by patshaughnessy** [comment]
I just now rebased and re-applied the database storage changes - just in time for Christmas!

Haven't had time to test it with rails3.2.rc yet... if you do try it, let me know what happens.

Thoughtbot has a fantastic test suite, and I added tests and features for database storage, so I'll try to merge their changes into this fork more often and re-run the test suite... maybe once a month or so.
